deploy/olm/assemble/metadata-common.yaml

```
@@ -429,7 +429,7 @@ metadata:
         }
       ]
   relatedImages:
-    - image: docker.io/rook/ceph:master
+    - image: rook/ceph:master
```

```suggestion
    - image: rook/ceph:master
```

it was copied from what ocs-operator had any issue with previously or it is just an improvement

---

Just an improvement, technically all these updates were handled by the csv-merger tool in ocs-operator. But, here we are updating via `sed` command so made it similar to the other image field used in deployement spec such that both the fields are updated during the csv-gen.